samples/outlook/01-compose-basics/get-item-subject.yaml

```diff
@@ -0,0 +1,48 @@
+id: outlook-01-compose-basics-get-item-subject
+name: Get Item Subject
+description: Gets the subject of a compose item
+host: OUTLOOK
+api_set: {}
+script:
+    content: |
+        $("#run").click(run);
+
+
+        function run() {
+            let item = Office.context.mailbox.item as Office.ItemCompose;
+            item.subject.getAsync((result: Office.AsyncResult) => {
+                if (result.status !== Office.AsyncResultStatus.Succeeded) {
+                    OfficeHelpers.UI.notify(`Action failed with message ${result.error.message}`);
+                } else {
```

Comment:
What about just `return`-in out within the `if` statement.  And then you don't need the `else` and the nesting here.  Ditto for the next samples.